docs/source/system_replication/system_replication.rst

```diff
@@ -0,0 +1,114 @@
+==================================================
+USB-Based System Replication
+==================================================
+
+.. contents:: Table of Contents
+   :depth: 2
+   :local:
+
+Introduction
+============
+
+The `USB-based system replication` feature allows saving the full disk image of an x64
+based NI Linux RT capable target and applying it to same or different target(s) of the
+same model.
+
+Safemode, runmode contents, drivers, can all be replicated.
+
+Some settings such as Hostname, SSH keys, Salt Minion ID, that uniquely identify a
+device on a network are not replicated to prevent conflicts on the network.
+
+Creating USB drive
+==================
+
+NI Linux RT System Image ships with a `nilrt-recovery-media-x64.iso`. Version 24.8 and
+above can be used to create a USB drive capable of system replication.
+
+Format a USB drive with **FAT32** File System and set **NIRECOVERY** as Volume label.
+
+   .. image:: media/format_usb.png
+
+Mount `nilrt-recovery-media-x64.iso` found under
+`C:\\Program Files (x86)\\National Instruments\\RT Images\\Utilities\\Linux RT Safemode\\<version 24.8 and above>`
+and copy the contents to the USB drive.
+
+Getting Image
+============
+
+To take system image of an x64 NI Linux RT system, plug the USB drive and change boot
+order to boot into it.
```

Comment:
You can use the USB drive prepared above to make a system image of an x64 NI Linux RT target.

To make a system image, connect the USB drive to the NI Linux RT target, reboot the target, and boot to the USB drive. You might need to change the boot order in order to boot from the USB drive.

Reply:
Slightly reworded